Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/EditorMaterialSystemComponent.h

```diff
@@ -100,6 +102,9 @@ namespace AZ
             // AzToolsFramework::ToolsApplicationNotificationBus::Handler overrides...
             void AfterEntitySelectionChanged(const AzToolsFramework::EntityIdList& newlySelectedEntities, const AzToolsFramework::EntityIdList&) override;
 
+            // AzToolsFramework::ActionManagerRegistrationNotificationBus::Handler...
+            void OnMenuBindingHook() override;
+
             void PurgePreviews();
 
             QAction* m_openMaterialEditorAction = nullptr;
```

Comment:
Since launching these tools is using the action manager now, can the action pointers be removed?

Reply:
Yes, I did in a follow up commit :) good catch!